src/Core/src/Primitives/GridLength.cs

```diff
@@ -9,22 +9,16 @@ namespace Microsoft.Maui
 	public struct GridLength
 	{
 		/// <include file="../../docs/Microsoft.Maui/GridLength.xml" path="//Member[@MemberName='Auto']/Docs" />
-		public static GridLength Auto
-		{
-			get { return new GridLength(1, GridUnitType.Auto); }
-		}
+		public static readonly GridLength Auto = new GridLength(1, GridUnitType.Auto);
```

Comment:
Actual breaking change, from static property to static field, require recompile

Reply:
If we didn't want to make a breaking change here, we could accomplish the same thing by using a property with an initializer:

```C#
public static GridLength Auto { get; } = new GridLength(1, GridUnitType.Auto);
```